Fal 2020 community calls (#306)
USRSE/usrse.github.io: _events/2020/2020-09-community-call.md

---

> title: Fall Community Call

Reviewer: ```suggestion
title: September 2020 Community Call
```

expires: 2020-09-11
event_date: "September 10, 2020"
layout: event
repeated: false
---

The next community call is on September 10 11ET/10CT/9MT/8PT, the main topic of discussion is "Hiring and interviewing for RSE positions".

Some potential topics include: 
  - What qualities are hiring managers looking for?
  - What can RSEs do to better distinguish themselves in the process?
  - How do RSEs want to be evaluated? 
  - What have been some good/bad interview experiences?
  - What advice would you give to someone looking to land their dream RSE job?

Other topics are also welcome!  Bring them up on the call or on Slack.

> The Zoom connection details and a link to the draft US-RSE Communications Guide is posted in the general channel on Slack.   

Reviewer: ```suggestion
The Zoom connection details are posted in the general channel on Slack.   
```
